diff --git a/node_modules/nuxt-vite/dist/nuxt-vite.js-vite.js b/node_modules/nuxt-vite/dist/nuxt-vite.js-vite.js
index d96ee1e..fe05d9e 100644
--- a/node_modules/nuxt-vite/dist/nuxt-vite.js-vite.js
+++ b/node_modules/nuxt-vite/dist/nuxt-vite.js-vite.js
@@ -44,7 +44,7 @@ async function buildClient(ctx) {
       }
     },
     plugins: [
-      vitePluginVue2.createVuePlugin()
+      vitePluginVue2.createVuePlugin({ include: [/\.vue$/, /\.md$/] })
     ],
     server: {
       middlewareMode: true
@@ -93,7 +93,7 @@ const APP_TEMPLATE = `
 async function buildServer(ctx) {
   const _env = process.env.NODE_ENV;
   process.env.NODE_ENV = "production";
-  const vuePlugin = vitePluginVue2.createVuePlugin();
+  const vuePlugin = vitePluginVue2.createVuePlugin({ include: [/\.vue$/, /\.md$/] });
   process.env.NODE_ENV = _env;
   const serverConfig = vite.mergeConfig(ctx.config, {
     define: {
